### PR TITLE
add sassy no js message

### PR DESCRIPTION
**Pull request by getfutureproof-admin**
Hi GT, thought this would be a fun addition, what do you think?

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -27,7 +27,7 @@
     <title>Events</title>
   </head>
   <body>
-    <noscript>You need to enable JavaScript to run this app.</noscript>
+    <noscript>No JavaScript? Shame! Shame! Shame!</noscript>
```
Comment (Gingertonic) [inline public/index.html:30]: Make this one sentence, please!

**Review by Gingertonic — CHANGES_REQUESTED** [review]
Haha! Hilarious. Slight change only.

File: public/index.html (hunk 1 above)
Comment (getfutureproof-admin) [reply public/index.html]: Done!

File: public/index.html [hunk 2]
```diff
@@ -27,7 +27,7 @@
     <title>Events</title>
   </head>
   <body>
-    <noscript>No JavaScript? Shame! Shame! Shame!</noscript>
+    <noscript>No JavaScript... shame, shame, shame!</noscript>
```
Comment (Gingertonic) [inline public/index.html:30]: Perfect, thanks

**Review by Gingertonic — APPROVED** [review]

**Pull request merged**